CRM/Activity/Form/Search.php

```diff
@@ -73,37 +75,14 @@ public function preProcess() {
 
     $this->_done = FALSE;
 
-    $this->loadStandardSearchOptionsFromUrl();
-
-    // get user submitted values
-    // get it from controller only if form has been submitted, else preProcess has set this
-    if (!empty($_POST) && !$this->controller->isModal()) {
-      $this->_formValues = $this->controller->exportValues($this->_name);
-    }
-    else {
-      $this->_formValues = $this->get('formValues');
-
-      if ($this->_force) {
-        // If we force the search then merge form values with url values
-        // and set submit values to form values.
-        // @todo this is not good security practice. Instead define the fields in metadata & use
-        // getEntityDefaults.
-        $this->_formValues = array_merge((array) $this->_formValues, CRM_Utils_Request::exportValues());
-        $this->_submitValues = $this->_formValues;
-      }
-    }
+    parent::preProcess();
 
     if (empty($this->_formValues)) {
```

Comment:
The parent fn doesn't seem to do these rows - but should

Reply:
Agreed on that